src/middlewared/middlewared/plugins/pool_/pool.py

```diff
@@ -680,6 +697,30 @@ async def do_create(self, job, data):
         self.middleware.send_event('pool.query', 'ADDED', id=pool_id, fields=pool)
         return pool
 
+    @private
+    async def validate_dedup_table_quota(self, data, verrors, schema='pool_create'):
+        if data.get('dedup_table_quota') != 'CUSTOM' and data.get('dedup_table_quota_value') is not None:
```

Comment:
Instead of doing `data.get` 4 times on the same variables. Why not `data.get` only 2 times and store in variables and then do checks?